src/Cli/dotnet/commands/dotnet-test/LocalizableStrings.resx

```diff
@@ -217,4 +226,39 @@ Outputs a 'Sequence.xml' file in the current directory that captures the order o
   <data name="IgnoredArgumentsMessage" xml:space="preserve">
     <value>The following arguments have been ignored : "{0}"</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="CmdBlameCrashCollectAlwaysDescription" xml:space="preserve">
+    <value>Enables collecting crash dump on expected as well as unexpected testhost exit.</value>
+  </data>
+  <data name="CmdBlameCrashDescription" xml:space="preserve">
+    <value>Runs the tests in blame mode and enables collecting crash dump when testhost exits unexpectedly. 
+This option is currently only supported on Windows, and requires procdump.exe and procdump64.exe to be available in PATH.
+Or PROCDUMP_PATH environment variable to be set, and point to a directory that contains procdump.exe and procdump64.exe. 
+The tools can be downloaded here: https://docs.microsoft.com/en-us/sysinternals/downloads/procdump 
+Implies --blame.</value>
+  </data>
+  <data name="CmdBlameCrashDumpTypeDescription" xml:space="preserve">
+    <value>The type of crash dump to be collected. Implies --blame-crash.</value>
+  </data>
+  <data name="CmdBlameHangDescription" xml:space="preserve">
+    <value>Run the tests in blame mode and enables collecting hang dump when test exceeds the given timeout. Implies --blame-hang.</value>
```

Comment:
This causes `dotnet test --help` to print:

> `--blame-hang                             Run the tests in blame mode and enables collecting hang dump when test exceeds the given timeout. Implies --blame-hang.`

This is self-referential and redundant. Should `Implies --blame-hang.` be part of the documentation for `CmdBlameHangTimeoutDescription`?

Reply:
That's a typo, should have been: Implies `--blame`

--blame is the option that observes which tests are running but does not create any dumps.